internal/manifests/console/consolelink.go

```diff
@@ -71,6 +73,25 @@ func CreateOrUpdateConsoleLink(ctx context.Context, c client.Client, cl *console
 	return nil
 }
 
+func DeleteKibanaConsoleLink(ctx context.Context, c client.Client) error {
+
+	current := &consolev1.ConsoleLink{}
+	key := client.ObjectKey{Name: KibanaConsoleLinkName}
+	if err := c.Get(ctx, key, current); err != nil {
```

Comment:
I don't believe there is need to "get" in order to delete

Reply:
Why not?
`c.Delete(ctx, current)` takes a k8s object that exits in the cluster, in our case the ConsoleLink

Reply:
@btaani he means to say that it is not required to `get` first and then `delete`. You can directly delete by passing the `key`.

Reply:
Alright it worked, but even though consolelink was deleted, the `c.Delete()` function returned an error